src/components/File.vue

```diff
@@ -0,0 +1,71 @@
+<template>
```

Comment:
I realize that I'm missing the computed property here, but ran out of time.

I'd have to change the component and do a filter function on the tags attribute.